tests/Product_type_unit_test.py

```diff
@@ -0,0 +1,72 @@
+from selenium import webdriver
+import unittest
+import re
+import sys
+
+
+class ProductTest(unittest.TestCase):
+    def setUp(self):
+        # change path of chromedriver according to which directory you have chromedriver.
+        self.driver = webdriver.Chrome('/home/dr3dd/gsoc/chromedriver')
```

Comment:
For purposes of testing with Travis, the chromedriver needs to be in the same directory as the file being tested. See the new unit test example for reference.